public/sounds/applause.mp3

```diff
@@ -0,0 +1 @@
+placeholder
```

Comment:
instead of all these placeholders, can we get and add a publicly available sound which is appropriate.

this comment also applies to all the other placeholder mp3s in this pr